catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java

```diff
@@ -1371,6 +1389,210 @@ public void testQuerySendsHitsRequestBeforeResultsRequest() throws Exception {
+      assertFeature(getFeatureType, true, MOCK_TEMPORAL_SORT_PROEPRTY, "ASC");
+    }
+  }
+
+  @Test
+  public void testSortingSortOrderDescending() throws Exception {
+    mapSchemaToFeatures(ONE_TEXT_PROPERTY_SCHEMA_PERSON, ONE_FEATURE);
+    setUpMocks(null, null, ONE_FEATURE, ONE_FEATURE);
+    final QueryImpl propertyIsLikeQuery =
+        new QueryImpl(builder.attribute(Metacard.ANY_TEXT).is().like().text("literal"));
+    setupMapper(
+        MOCK_TEMPORAL_SORT_PROEPRTY, MOCK_RELEVANCE_SORT_PROEPRTY, MOCK_DISTANCE_SORT_PROEPRTY);
+    source.setMetacardMappers(metacardMappers);
+    propertyIsLikeQuery.setSortBy(new SortByImpl(Result.TEMPORAL, SortOrder.DESCENDING));
+
+    final ArgumentCaptor<GetFeatureType> captor = ArgumentCaptor.forClass(GetFeatureType.class);
+    source.query(new QueryRequestImpl(propertyIsLikeQuery));
+    verify(mockWfs, times(2)).getFeature(captor.capture());
+
+    for (final GetFeatureType getFeatureType : captor.getAllValues()) {
+      assertFeature(getFeatureType, true, MOCK_TEMPORAL_SORT_PROEPRTY, "DESC");
+    }
+  }
+
+  @Test
+  public void testSortingDisabled() throws Exception {
+    mapSchemaToFeatures(ONE_TEXT_PROPERTY_SCHEMA_PERSON, ONE_FEATURE);
+    setUpMocks(null, null, ONE_FEATURE, ONE_FEATURE);
+    final QueryImpl propertyIsLikeQuery =
+        new QueryImpl(builder.attribute(Metacard.ANY_TEXT).is().like().text("literal"));
+    setupMapper(
+        MOCK_TEMPORAL_SORT_PROEPRTY, MOCK_RELEVANCE_SORT_PROEPRTY, MOCK_DISTANCE_SORT_PROEPRTY);
+    source.setMetacardMappers(metacardMappers);
+    source.setDisableSorting(true);
+    propertyIsLikeQuery.setSortBy(new SortByImpl(Result.TEMPORAL, SortOrder.ASCENDING));
+
+    final ArgumentCaptor<GetFeatureType> captor = ArgumentCaptor.forClass(GetFeatureType.class);
+    source.query(new QueryRequestImpl(propertyIsLikeQuery));
+    verify(mockWfs, times(2)).getFeature(captor.capture());
+
+    for (final GetFeatureType getFeatureType : captor.getAllValues()) {
+      assertFeature(getFeatureType, false, MOCK_TEMPORAL_SORT_PROEPRTY, "ASC");
+    }
+  }
+
+  @Test
+  public void testSortingNoSortMapping() throws Exception {
+    // if sorting is enabled but there is no sort mapping, throw an UnsupportedQueryException
+    expectedEx.expect(UnsupportedQueryException.class);
+    expectedEx.expectMessage("Source WFS_ID does not support specified sort property title");
+
+    mapSchemaToFeatures(ONE_TEXT_PROPERTY_SCHEMA_PERSON, ONE_FEATURE);
+    setUpMocks(null, null, ONE_FEATURE, ONE_FEATURE);
+    final QueryImpl propertyIsLikeQuery =
+        new QueryImpl(builder.attribute(Metacard.ANY_TEXT).is().like().text("literal"));
+    setupMapper(null, null, null);
+    source.setMetacardMappers(metacardMappers);
+    source.setDisableSorting(false);
+    propertyIsLikeQuery.setSortBy(new SortByImpl("title", SortOrder.ASCENDING));
+
+    final ArgumentCaptor<GetFeatureType> captor = ArgumentCaptor.forClass(GetFeatureType.class);
```

Comment:
Don't need a captor for this test (or for any of the following tests that expect an exception).

Reply:
looks like something I missed.  Cleaning it up